openfisca_us/variables/entity/person.py

```diff
@@ -16,6 +16,28 @@ class person_weight(Variable):
     definition_period = YEAR
 
 
+class is_tax_unit_head(Variable):
+    value_type = bool
+    entity = Person
+    label = "Head of tax unit"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        # Use order of input (first)
+        return person.tax_unit.members_position == 0
```

Comment:
is this based on the raw CPS structure?

Reply:
Not intentionally, just needed a sensible default value. It'll be overridden by input values.